_includes/description.md

```diff
@@ -1,3 +1,5 @@
+## Description
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5607142)